docs/api/conf.py

```diff
@@ -72,6 +72,7 @@
 html_copy_source = False
 
 html_title = 'Inrupt {0} Documentation'.format(name)
+html_favicon= "https://docs.inrupt.com/inrupt_stickers_v2-03.png"
```

Comment:
Can this be part of the theme's public directory? That way we're not relying on a URL being maintained outside of normal deployment of the docs.